src/server/RHData.py

```diff
@@ -852,6 +852,19 @@ def get_pilot_attribute(self, pilot_or_id, name):
 
         return self._Database.PilotAttribute.query.filter_by(id=pilot_id, name=name).one_or_none()
 
+    def get_pilot_attribute_value(self, pilot_or_id, name, default_value=False):
+        if isinstance(pilot_or_id, Database.Pilot):
+            pilot_id = pilot_or_id
+        else:
+            pilot_id = self._Database.Pilot.query.get(pilot_or_id).id
+
+        attr = self._Database.PilotAttribute.query.filter_by(id=pilot_id, name=name).one_or_none()
+
+        if attr:
+            return attr.value
+        else:
+            return default_value
+
```

Comment:
You could use a ternary if operator here. However I question the usefulness of the method, and it seems not to be used anywhere. Consider ditching it, and replace possible uses with 

```
get_pilot_attributes()['name']
get_pilot_attributes().get('name', default)
```
Same for `get_pilot_attribute`